vcpkg/reference/installation-tree-layout.md

```diff
@@ -0,0 +1,119 @@
+| [`tools/<port>`](#layout-tools)                           | Executable tools                                                |
+
+
+### <a name="layout-bin"></a> `bin` and `debug/bin` directories
+
+On Windows, these directories contain DLL and PDB files for release and debug configuration
+respectively. Any executable file produced by a port should be moved to a `tools/<port>` directory.
+
+### <a name="layout-include"></a> `include`
+
+Contains header files (`.h`, `.hpp`, `.hxx`), the internal layout of this directory should reflect
+upstream usage of the package's header files. For example, a `contoso` library that provides
+`contoso/contoso.h` should create the necessary subdirectories to reflect the relative path, in this
+case, the header file should be placed in `include/contoso/contoso.h`.
+
+vcpkg forbids installing some reserved header file names in the root of the `include` directory, for
+example: `err.h`, `user.h`, `time.h`, and others. Libraries that provide a forbidden header file name should
+place their header files inside a `include/<port>` directory. If the library intends to replace a
+system header file, it should set the `VCPKG_POLICY_ALLOW_RESTRICTED_HEADERS` policy in its
+`portfile.cmake`.
+
+### <a name="layout-lib"></a> `lib` and `debug/lib` directories
+
+Contains static libraries, import libraries (on Windows), and shared libraries (on non-Windows).
+configuration-dependent directories.
+
+### <a name="layout-manual-link"></a> `lib/manual-link` and `debug/lib/manual-link` directories
+
+Contains object files that must be manually linked, these directories are configuration dependent.
+Each port must make sure to place manually-linkable files in these directories explicitly.
+
+Files that may cause issues when linked automatically must be placed in the `lib/manual-link`
+folders instead of the `lib` directory. For example, if a library is intended to define the
+`main()` function for a program. 
+
+### <a name="layout-pkgconfig"></a> `lib/pkgconfig` and `share/pkgconfig` directories
+
+Contains pkgconfig integration files (`.pc`).
+
+### <a name="layout-share"></a> `share/<port>`
+
+Contains miscellaneous files installed by each port, at the very least, each port is required to
+provide a `LICENSE` file in `share/<port>` (replace `<port>` with the port name). Some examples of
+files contained in this directory are: CMake integration files,
+[usage](../maintainers/handling-usage-files.md) files, SPDX files, etc.
+
+### <a name="layout-copyright"></a> `share/<port>/copyright`
+
+vcpkg expects ports to provide a `copyright` file that contains the license information for the
+installed package. See the [maintainer
+guide](../contributing/maintainer-guide.md#install-copyright-file) for more information.
+
+### <a name="layout-cmake"></a> `share/<package>/<package>Config.cmake`, `share/<package>/<package>-config.cmake`
```

Comment:
```suggestion
### <a name="layout-cmake"></a> `share/<lower-case-package>/<package>Config.cmake`, `share/<package>/<package>-config.cmake`
```
... if lower-case dirs in vcpkg are preferred, to ensure package config consistently lands in port dir when possible.

(Case would be less relevant if vcpkg would decide to generally put CMake config into `/share/cmake`, ensuring different space for general port data and for cmake config. `<prefix>/share/cmake/<name>*/` is in the search path even before `<prefix>/share/<name>*/`.)